src/route.js

```diff
@@ -10,12 +10,13 @@ const ActivityWindow = () => import('./views/activity/ActivityWindow.vue');
 const ActivityBrowser = () => import('./views/activity/ActivityBrowser.vue');
 const ActivityEditor = () => import('./views/activity/ActivityEditor.vue');
 
+const Settings = () => import('./views/settings/Settings.vue');
+
```

Comment:
Why move the line? Will there be more routes?

Reply:
Oh sorry. First I thought so but then I decided not to. I'll revert it.